backend/src/main/java/com/bakdata/conquery/models/auth/web/AuthorizationExceptionMapper.java

```diff
@@ -19,9 +18,9 @@
 	@Override
 	public Response toResponse(AuthorizationException exception) {
 		log.warn("Shiro failed to authorize the request. See the following trace:", exception);
-		return Response.status(Response.Status.UNAUTHORIZED)
+		return Response.status(Response.Status.FORBIDDEN)
 			.type(MediaType.APPLICATION_JSON_TYPE)
-			.entity(new ErrorMessage(Response.Status.FORBIDDEN.getStatusCode(), "Your error has been logged"))
+			.entity("Not sufficient permissions to perform action: " + exception.getMessage())
```

Comment:
Is in der Exception der ganze Stacktrace drin? Erscheint mir etwas riskant das einfach rauzugeben

Reply:
Nein da ist lediglich drinne, an welcher berechtigung es scheitert